dev/docker/up-add-node-monitoring-to-up

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
How and when does this script get run?

Reply:
manually if you want to start all the required elements.